cirq-google/cirq_google/api/v2/device.proto

```diff
@@ -30,12 +33,55 @@ message DeviceSpecification {
   string developer_recommendations = 4;
 }
 
+// Replaces `GateDefinition`.
```

Comment:
Maybe also explain the purpose of this message in addition to explaining what it is replacing (which will become increasingly obsolete over time)